R/plot_jags_tracedx.R

```diff
@@ -37,14 +37,12 @@
 #'
 #'   data <- serodynamics::nepal_sees_jags_post
```

Comment:
formatting is off here; move examples to separate file?
```suggestion
#' data <- serodynamics::nepal_sees_jags_post
```

Reply:
on second look, the indentation here is correct; it's the indentation later in the example that seems off. Either way, if we move the example to an external file the linter will check it automatically and https://github.com/r-lib/styler or https://github.com/posit-dev/air can fix it for you.

Reply:
Moved example to its own file